src/main/java/cn/nukkit/block/BlockSnowLayer.java

```diff
@@ -5,21 +5,34 @@
 import cn.nukkit.item.ItemSnowball;
 import cn.nukkit.item.ItemTool;
 import cn.nukkit.level.Level;
+import cn.nukkit.math.AxisAlignedBB;
 import cn.nukkit.math.BlockFace;
 import cn.nukkit.utils.BlockColor;
 
 /**
  * Created on 2015/12/6 by xtypr.
  * Package cn.nukkit.block in project Nukkit .
  */
-public class BlockSnowLayer extends BlockFlowable {
+public class BlockSnowLayer extends BlockFallable {
+
+    private int meta;
 
     public BlockSnowLayer() {
         this(0);
     }
 
     public BlockSnowLayer(int meta) {
-        super(meta);
+        this.meta = meta;
```

Comment:
Why are we storing meta here? It will cause other methods to return incorrect values.

Reply:
The meta is for multiple snow layers placed top of each other, its here because BlockFallable doesnt extends BlockMeta

Reply:
Ok